docs/contributing.md

```diff
@@ -1,6 +1,5 @@
 # How to Contribute
 
-
```

Comment:
nit: the formatting changes would ideally go into its own commit (not sure if you even intended them)

Reply:
Ugh, vscode autoformatting again. I'll try to rip this out.